environment.yml

```diff
@@ -12,5 +12,5 @@ dependencies:
 - pip
 - pip:
   - openfermion>=1.5
-  - qibo
-  - pyscf
+  - qibo=0.1.10

```

Comment:
Why the restriction on the Qibo version?

Reply:
I agree with @chmwzc let's do what qibotn team did. Also restriction on Qibo version above has to be >= or ==. But if there is no need for a restriction, it should be remove.

Reply:
Should've added a > there, just adding it as a reminder of the lowest working version.
Anyway, I agree with you since this dependency issue is out of the scope of the branch.